README.md

```diff
@@ -235,6 +267,46 @@ This property either contains an absolute or relative path of the parent mod or
 
 ---
 
+## The `"language"` Type
+
+#### The `"language.code"` Property
+
+**Level:** **REQUIRED**
+
+**Data Type**: `string`
+
+**Data Semantics**: Language Code
+
+**Description:**
+
+This property holds an [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two letter language code.
```

Comment:
That's something I do like, because that could tie in with the available language codes in the `PG.StarWarsGame.Localisation` library, making it easier to match the `*.dat` file with the language and keeping it in sync with the mod metadata.

I'm currently using the C# Culture codes there though, so I'll have to look into how finicky that would be to match.